test/unit/DrbbResult.test.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-env mocha */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { spy } from 'sinon';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+
+import DrbbResult from './../../src/app/components/Drbb/DrbbResult';
+import { mockRouterContext } from '../helpers/routing';
+import workData from '../fixtures/work-detail.json';
+
+describe('DrbbResult', () => {
+  let component;
+  const context = mockRouterContext();
+
+  describe('with work prop', () => {
+    const authors = workData.data.agents.filter(agent => agent.roles.includes('author'));
+    before(() => {
+      component = shallow(<DrbbResult work={workData.data}/>);
+    });
+
+    it('should be wrapped in a .drbb-item class', () => {
+      expect(component.find('.drbb-item')).to.be.defined;
+      expect(component.find('.drbb-item')).to.have.length(1);
+    });
+
+    it('should have a link with .drbb-result-title class', () => {
+      expect(component.find('.drbb-result-title')).to.be.defined;
+      expect(component.find('.drbb-result-title')).to.have.length(1);
+    });
+
+    it('should have links to authors', () => {
+      expect(component.find('.drbb-result-author')).to.be.defined;
+      expect(component.find('.drbb-result-author')).to.have.length(authors.length);
```

Comment:
Since there's a bunch of author logic, I'd check for that too - so instead of having a length, check for authors.text(), just to make sure stuff like comma separation, etc is working.  You also can then check for the fallback if there are no authors